examples/ex_5.rs

```diff
@@ -38,8 +38,8 @@ fn main() {
 
   /* Print a border around the main window */
   box_(my_menu_win, 0, 0);
-  mvprintw(LINES - 3, 0, "Press <ENTER> to see the option selected");
-  mvprintw(LINES - 2, 0, "F1 to exit");
+  mvprintw(LINES() - 3, 0, "Press <ENTER> to see the option selected");
+  mvprintw(LINES() - 2, 0, "F1 to exit");
```

Comment:
LINES is a function - the compiler was complaining that you cannot subtract from a function.